src/test/resurces/ddo.json

```diff
@@ -25,10 +25,10 @@
 
   "created": "2002-10-10T17:00:00Z",
```

Comment:
check the path for this file, you miss letter:
src/test/res(**o**)urces/ddo.json

Reply:
done